addons/transform/org.openhab.transform.jinja/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,28 @@
+Manifest-Version: 1.0
+Bundle-ClassPath: .,
+ lib/jinjava-2.4.12.jar,
+ lib/jsoup-1.11.3.jar,
+ lib/re2j-1.2.jar,
+ lib/jackson-core-2.9.8.jar,
+ lib/jackson-databind-2.9.8.jar,
+ lib/jackson-annotations-2.9.8.jar,
+ lib/commons-lang3-3.8.1.jar
```

Comment:
The dependency list is just quite huge, but there is nothing we can do about that, I guess? Maybe you are only using a subset of "jinjava" and we can reduce this list?

Reply:
I checked again and all are needed to run the simple test examples :-(